docs/modules/ROOT/pages/support.adoc

```diff
@@ -4,6 +4,25 @@ Please use our official link:https://issues.couchbase.com/projects/CMOS/issues[J
 
 No official support is currently provided but best efforts will be made and we are keen to hear of any issues.
 
+== Diagnostics
+
+All components of CMOS log their output to `/logs/<component>.log` inside the container.
+By default, they will also log to standard output, which you can access using `docker logs`, `kubectl logs`, or similar.
+This can be disabled by setting the environment variable `LOG_TO_STDOUT` to `false` when starting CMOS.
+
+=== Collecting Information
+
+Use the link:/collectinfo.html[Collect Information] form to create a tar-ball of all CMOS logs and configuration.
```

Comment:
Oh nice, does this work? Might have to update the others as was firing through localhost:8080

Reply:
That works - the link takes you to `/collectinfo.html`, where there's a little bit of HTML kludgery to POST to `/config/api/v1/collectInformation`.

Reply:
OK cool, need to update the tooling/config ones in nav